src/isar_robot/robotinterface.py

```diff
@@ -131,36 +102,6 @@ def get_telemetry_publishers(
         )
         publisher_threads.append(pose_thread)
 
-        battery_publisher: MqttTelemetryPublisher = MqttTelemetryPublisher(
-            mqtt_queue=queue,
-            telemetry_method=self._get_battery_telemetry,
-            topic=f"isar/{isar_id}/battery",
-            interval=5,
-            retain=False,
-        )
-        battery_thread: Thread = Thread(
-            target=battery_publisher.run,
-            args=[isar_id, robot_name],
-            name="ISAR Robot Battery Publisher",
-            daemon=True,
-        )
-        publisher_threads.append(battery_thread)
-
-        pressure_publisher: MqttTelemetryPublisher = MqttTelemetryPublisher(
-            mqtt_queue=queue,
-            telemetry_method=self._get_pressure_telemetry,
-            topic=f"isar/{isar_id}/pressure",
-            interval=5,
-            retain=False,
-        )
-        pressure_thread: Thread = Thread(
-            target=pressure_publisher.run,
-            args=[isar_id, robot_name],
-            name="ISAR Robot Pressure Publisher",
-            daemon=True,
-        )
-        publisher_threads.append(pressure_thread)
-
```

Comment:
Why do we want to remove the battery and pressure values?

Reply:
Just reverting this now